model_api/python/model_api/models/action_classification.py

```diff
@@ -0,0 +1,220 @@
+"""
+ Copyright (C) 2024 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+import numpy as np
+from model_api.adapters.utils import RESIZE_TYPES, InputTransform
+
+from .model import Model
+from .types import BooleanValue, ListValue, NumericalValue, StringValue
+from .utils import ClassificationResult, load_labels
+
+if TYPE_CHECKING:
+    from model_api.adapters.inference_adapter import InferenceAdapter
+
+
+class ActionClassificationModel(Model):
```

Comment:
Overall, I thinks we need also a method which can tell users which is the optimal clips size. If user passes a shorter clip, it can be extended with empty frames.
Otherwise, the model is going to have dynamic input shape, which is not intended I believe

Reply:
I think it's better to raise an error which tells "need to set dynamic input shape" when user gives inputs w/ various shapes although user gives OV IR w/ fixed input shape.

Reply:
I'm ok with that. OV's error in that case could be not quite verbose